### PR TITLE
Changed the markup of homepage, Changed structure of the layouts, add…

### PR DESCRIPTION
**Pull request by srikant-code**
Added many SVG assets that's why showing 37 files changed. 
The other changes are same as the PR/83s.
@arkajyotiMukherjee @S-ayanide 

Signed-off-by: Srikant Sahoo <srikantsahoo3601@gmail.com>

### CODE REVIEW

File: src/components/button/button.tsx [hunk 1]
```diff
@@ -8,17 +8,22 @@ interface IButton {
 
 // Default Button Styles
 const Button = styled.button<IButton>`
-  height: 3rem;
-  min-width: 10rem;
-  width: ${props => (props.screen === "small" ? "70%" : "15rem")};
+  height: 4.5rem;
+  min-width: 8rem;
+  width: ${props => (props.screen === "small" ? "50%" : "15rem")};
```
Comment (S-ayanide) [inline src/components/button/button.tsx:13]: By changing the width to 50% on small screens you are going against the design which almost takes 80% of the whole screen width.

File: src/components/button/button.tsx [hunk 2]
```diff
@@ -8,17 +8,22 @@ interface IButton {
...
   border: none;
+  outline:none;
   background: ${props =>
     props.gradientColor === "purple"
       ? props.theme.gradient.purple
       : props.theme.gradient.green};
   color: ${props => props.theme.colors.pureWhite};
-  border-radius: 0.25rem;
-  font-size: 0.9rem;
+  display:flex;
```
Comment (S-ayanide) [inline src/components/button/button.tsx:21]: What is the purpose of putting flex row in buttons? Are you planning to add icons or more than one child inside of it?

File: src/components/button/button.tsx [hunk 3]
```diff
@@ -30,12 +35,18 @@ const WhiteOnGreenButtonStyles = styled.button`
...
   border: none;
+  outline:none;
   border-radius: 0.2rem;
   background: ${props => props.theme.colors.pureWhite};
   color: ${props => props.theme.colors.greenDark};
-  font-size: 0.7rem;
+  font-size: ${props => props.theme.fontsize.paragraph};
   font-weight: bold;
   cursor: pointer;
+  width: 208px;
```
Comment (S-ayanide) [inline src/components/button/button.tsx:45]: Try not to put px but use rems, also don't hardcode the width until you are absolutely sure you would want this button to have a fixed width everywhere, else provide the width as props, would be more flexible.

File: src/components/button/button.tsx (hunk 1 above)
Comment (srikant-code) [reply src/components/button/button.tsx:13]:
```suggestion
  width: ${props => (props.screen === "small" ? "60%" : "15rem")};
```
image

Actually see this. In small screens, it seems no way 70% of the width. That's why changed it to 50%, but this can be 55% or 60% as well to make it perfect.

File: src/components/button/button.tsx (hunk 2 above)
Comment (srikant-code) [reply src/components/button/button.tsx:21]: I actually added flex when I was testing these out👇 since it was distorted.
```
vertical-align:middle;
text-align:center;
```
Which didn't worked. So tried using flex.
But yes removing that works fine as of now.
```suggestion
  display:flex;     //remove this line and the next 3 lines as well.
```
 justify-content:center;
  align-items:center;
  flex-flow:row wrap;

Comment (arkajyotiMukherjee) [reply src/components/button/button.tsx:21]: yes remove these flex related properties

File: src/components/button/button.tsx [hunk 4]
```diff
@@ -30,12 +35,18 @@ const WhiteOnGreenButtonStyles = styled.button`
...
+  height: 33px;
+  background: #FFFFFF;
```
Comment (arkajyotiMukherjee) [inline src/components/button/button.tsx:47]: use theme colors only

File: src/components/containers/CloudNativeWay.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+      )
+    }
+  };
+
+const CloudNativeWay: React.FC = () => {
+    return (
+        <Section>
+            <CloudNativeWayDiv>
+                <Heading>Do it the cloud-native way</Heading>
+                <>
```
Comment (arkajyotiMukherjee) [inline src/components/containers/CloudNativeWay.tsx:46]: a fragment is not required here
read more about fragments here: https://reactjs.org/docs/fragments.html

File: src/components/containers/GetStartedWithLitmus.tsx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+const Width100 = styled.div`
+    width:100%;
+    display:flex;
+    flex-flow:wrap column;
+    justify-content:center;
+    align-items:center;
+`
+
+// co React.FC = ({ children }) => {
```
Comment (arkajyotiMukherjee) [inline src/components/containers/GetStartedWithLitmus.tsx:39]: large sections of commented code shouldn't be given as in PR

File: src/components/containers/SeeWhatOur.tsx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    display:flex;
+    flex-flow:wrap column;
+    justify-content:center;
+    align-items:start;
+`
+const Rowflex = styled.div`
+    width:100%;
+    display:flex;
+    flex-flow:wrap row;
+    justify-content:start;
```
Comment (arkajyotiMukherjee) [inline src/components/containers/SeeWhatOur.tsx:28]: justify-content start and align-items start are default properties

File: src/components/containers/StartInMinutes.tsx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+    height: 2.7rem;
+    background: #109B67;
+    border: 8px solid #FFFFFF;
+    border-radius:50%;
+    box-shadow: 0px 9px 11px rgba(91, 68, 186, 0.15);
+`
+
+const Width100 = styled.div`
+    min-width:100%;
+    width:100%;
```
Comment (arkajyotiMukherjee) [inline src/components/containers/StartInMinutes.tsx:27]: min-width and width together doesn't make sense

File: src/components/containers/StartInMinutes.tsx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+const Columnflex = styled.div`
+    width:50%;
+    display:flex;
+    flex-flow:wrap column;
+    justify-content:center;
+    align-items:start;
+`
+
+// co React.FC = ({ children }) => {
```
Comment (arkajyotiMukherjee) [inline src/components/containers/StartInMinutes.tsx:42]: remove commented code sections

File: src/components/layout/responsive-row.tsx [hunk 10]
```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
   ${({ theme }) =>
     theme.screens.sm
       ? css`
-          flex-direction: column;
+          flex-flow: column;
```
Comment (arkajyotiMukherjee) [inline src/components/layout/responsive-row.tsx:10]: unnecessary change...flex-flow without flow property is as good as flex-direction

File: src/components/layout/responsive-row.tsx [hunk 11]
```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
...
           align-items: start;
         `
       : css`
-          flex-direction: row;
+          flex-flow: row;
```
Comment (arkajyotiMukherjee) [inline src/components/layout/responsive-row.tsx:14]: remove flex flow and add flex direction

File: src/components/layout/responsive-row.tsx [hunk 12]
```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
...
+          align-items: center;
```
Comment (arkajyotiMukherjee) [inline src/components/layout/responsive-row.tsx:15]: the design doesn't require it to be aligned in center

File: src/components/layout/responsive-row.tsx [hunk 13]
```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
...
           justify-content: center;
         `};
 `;
 
+const PanelContainerMD = styled.div`
```
Comment (arkajyotiMukherjee) [inline src/components/layout/responsive-row.tsx:20]: this component is unnecessary we can modify the responsive row to suit all these needs

File: src/components/layout/responsive-row.tsx [hunk 14]
```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
...
+      `
+    };
+`;
+
 const ResponsiveRow: React.FC = ({ children }) => {
   return <PanelContainer>{children}</PanelContainer>;
 };
 
-export { ResponsiveRow };
+const ResponsiveRowMD: React.FC = ({ children }) => {
```
Comment (arkajyotiMukherjee) [inline src/components/layout/responsive-row.tsx:46]: this is also not required

File: src/components/nav/Nav.tsx [hunk 15]
```diff
@@ -23,14 +23,18 @@ const LogoDiv = styled.div`
...
+  height: fit-content;  
   background: ${props => props.theme.colors.spaceGrey};
-  padding: 0.5rem 0.5rem;
+  padding: 0.3rem 0.5rem;
+  padding-top: 0.5rem;
   margin-left: 1rem;
   margin-top: 0.6rem;
-  box-shadow: -0.2rem 0.3rem 0.7rem 0.1rem rgba(0, 0, 0, 0.4);
-  border-radius: 0.5rem;
+  width: 72px;
```
Comment (arkajyotiMukherjee) [inline src/components/nav/Nav.tsx:32]: don't use pixels....use rems instead

File: src/components/nav/Nav.tsx [hunk 16]
```diff
@@ -23,14 +23,18 @@ const LogoDiv = styled.div`
...
+  border: 1px solid #FFFFFF;
```
Comment (arkajyotiMukherjee) [inline src/components/nav/Nav.tsx:33]: take color from theme

File: src/styles/global.css [hunk 17]
```diff
@@ -1,7 +1,7 @@
 @import url("https://fonts.googleapis.com/css2?family=Ubuntu&display=swap");
 
 html {
-  font-family: "Ubuntu", sans-serif;
+  font-family: "Ubuntu", sans-serif !important;
```
Comment (arkajyotiMukherjee) [inline src/styles/global.css:4]: no other style is overwriting this....not required

File: src/styles/theme.tsx [hunk 18]
```diff
@@ -33,6 +33,11 @@ export default {
       "linear-gradient(135.47deg, #858CDD 1.77%, rgba(133, 140, 221, 0) 64.41%), #5B44BA;"
   },
   py: (value: number | string) =>
-    `padding-top: ${value}; padding-bottom: ${value};`
+    `padding-top: ${value}px; padding-bottom: ${value}px;`,
```
Comment (S-ayanide) [inline src/styles/theme.tsx:36]: You don't require pixels

File: src/components/section-layout/section.tsx [hunk 19]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
```
Comment (arkajyotiMukherjee) [inline src/components/section-layout/section.tsx:1]: can you explain the purpose of this component and why viewport widths and heights are used

File: src/components/button/button.tsx [hunk 20]
```diff
@@ -8,17 +8,22 @@ interface IButton {
...
 // Default Button Styles
 const Button = styled.button<IButton>`
-  height: 3rem;
-  min-width: 10rem;
-  width: ${props => (props.screen === "small" ? "70%" : "15rem")};
+  height: 4.5rem;
+  min-width: 8rem;
+  width: ${props => (props.screen === "small" ? "50%" : "15rem")};
   border: none;
+  outline:none;
```
[the remark this answers is not in the document]
Comment (srikant-code) [reply src/components/button/button.tsx:15]: No, the <button> always has an outline by default. So we need to specify it explicitly.

File: src/components/button/button.tsx (hunk 1 above)
Comment (srikant-code) [reply src/components/button/button.tsx:13]: I finally made it 60% instead 70% after testing out different values. Resolving this thread.

File: src/components/section-layout/section.tsx (hunk 19 above)
Comment (srikant-code) [reply src/components/section-layout/section.tsx:1]: There was a need for section component and I used it in almost all the places(And it is required). It is made to bind the content in the center of the page for larger screens. If you remove the section component then all the content will always be taking up 100% of the width of the page like the starter code had also for extra larger desktops. So this component is used to fix that issue.

File: src/styles/global.css (hunk 17 above)
Comment (srikant-code) [reply src/styles/global.css:4]: No, it is required. I don't know why but the Roboto font is by default on everything. And writing important fixed this. See the difference. You will get it.
image
image

File: src/components/layout/responsive-row.tsx (hunk 12 above)
Comment (srikant-code) [reply src/components/layout/responsive-row.tsx:15]: This is not required but it was useful in this case👇. And one thing to note adding that didn't changed other things :)
image
image

**Comment by srikant-code** [comment]
So I have implemented all these changes. I will push the new codes by today's night.

**Comment by srikant-code** [comment]
- I have pushed a new commit please review it. @arkajyotiMukherjee @S-ayanide .
- You may have many questions with the code but you should try running it to know the actual cause of writing those. The pages are not 100% responsive but yes I can say its 90% responsive`(for phones, tablets and laptops)`. Ultra large desktops have some issues pointed out in the next point.

- The issue is about the Navbar. I would have to restructure it. You can actually see the issue when you browse through `ultra-large desktops(>3000px)`.

- I have split the `WhyLitmus` and `Homepage` into several files for readability and abstracting codes to specific areas.

- One important thing to note. I have reduced both the page sizes to less than 150kb (overall including the chaos bird images) which will drastically boost the page loading time. I have done this by converting Shreyas Chaos bird huge PNGs into vectors in this weekend. That's why took me some time.

- I had also tried adding a click to copy area in the `Get started with Litmus` section but that seems not to work so may need help from you.

- The other sections(`Footer and Adopters`) will be added after our PRs get merged so that we can use those codes written by Mayukh.

**Comment by arkajyotiMukherjee** [comment]
Amazing job @srikant-code

**Comment by srikant-code** [comment]
This is a feature I added for faster accessibility of copying the install command. And I finally solved the bug. Will push this code later.
image
image

File: src/styles/global.css (hunk 17 above)
Comment (mallick370) [reply src/styles/global.css:4]: You can fix this issue by changing the font-family in global-styles.tsx to Ubuntu......there by default it has been kept sans-serif
